bin/git_functions.sh

```diff
@@ -936,6 +946,13 @@ repository_setup_release () {
         edit_deploy_templates "${_RELEASE}"
     fi
 
+    # Update the pinned version number to the current ${_RELEASE}
+    #+ for released Trraform modules
```

Comment:
Typos and stray '+'
```
# for released Terraform modules
```

Reply:
That is not a stray +, that is an old idiom to illustrate that the comment carries to the following line while being able to stay inline within an 80 char term limit.

Reply:
Really? I didn't know that idiom, consider me educated!

However, Terraform is still spelled Terraform and not Trraform ;-p